lib/hades/accounts/auth.ex

```diff
@@ -28,7 +28,7 @@ defmodule Hades.Accounts.Auth do
     end
   end
 
-  def reset_password(user, attrs) do
+  def update_password(user, attrs) do
```

Comment:
The concept here seems better using the verb "update". Reset would be a different feature when user forget his pass.

Reply:
👍